ajv/app.js

```diff
@@ -1,18 +1,17 @@
-const Ajv = require("ajv");
-const fs = require("fs");
-const glob = require("glob");
-const express = require("express");
+import Ajv2020 from "ajv/dist/2020.js";
+import fs from "fs";
+import glob from "glob";
+import express from "express";
+import Debug from "debug";
+
+const debug = Debug("ajv-schema-validator");
+
 const app = express();
-const debug = require("debug")("ajv-schema-validator");
-
-const ajv = new Ajv({
-  meta: false,
-  extendRefs: true,
-  unknownFormats: "ignore",
-  allErrors: false,
-  schemaId: "auto",
+
+const ajValidator = new Ajv2020({
+  allErrors: true,
+  strict: false,
```

Comment:
What errors will this raise if it was true?. One of the main changes is that we can't have unregistered keyword $refs but worth checking if that can be disabled independently.

Reply:
I am not sure I get this Mebin. Do you mind if we can have a quick chat on this please.

Reply:
As discussed made strict as true and included strictSchema

Reply:
@MebinAbraham - with `strict: true` it seems we are getting issues with the references For example - "Error: strict mode: missing type "string" for keyword "pattern"". Although the type has been defined as a part of the ref

Reply:
👍